  <font size = 6><center> Titanic Kaggle Competition Submission </center></font>
  <font size = 4><center> Gariman Singh </center></font>

https://www.kaggle.com/code/sgarimann/titanic-submission#Making-an-ensemble

In [1]:
%pip install tensorflow-decision-forests

  Using cached tensorflow_decision_forests-1.8.1.tar.gz (15.2 MB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
INFO: pip is looking at multiple versions of tensorflow-decision-forests to determine which version is compatible with other requirements. This could take a while.
Note: you may need to restart the kernel to use updated packages.


ERROR: Could not find a version that satisfies the requirement tensorflow~=2.15.0 (from tensorflow-decision-forests) (from versions: 2.16.0rc0, 2.16.1)
ERROR: No matching distribution found for tensorflow~=2.15.0


# Importing dependencies

In [2]:
import numpy as np
import pandas as pd
import os

import tensorflow as tf
import tensorflow_decision_forests as tfdf

KeyboardInterrupt: 

# Loading dataset

In [ ]:
train_df = pd.read_csv("train.csv")
serving_df = pd.read_csv("test.csv")

train_df.head(10)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   
5            6         0       3   
6            7         0       1   
7            8         0       3   
8            9         1       3   
9           10         1       2   

                                                Name     Sex   Age  SibSp  \
0                            Braund, Mr. Owen Harris    male  22.0      1   
1  Cumings, Mrs. John Bradley (Florence Briggs Th...  female  38.0      1   
2                             Heikkinen, Miss. Laina  female  26.0      0   
3       Futrelle, Mrs. Jacques Heath (Lily May Peel)  female  35.0      1   
4                           Allen, Mr. William Henry    male  35.0      0   
5                                   Moran, Mr. James    male   NaN      0   
6                            McCarthy, Mr. Timothy J    male  54.0      0   
7                     Palsson, Master. Gosta Leonard    male   2.0      3   
8  Johnson, Mrs. Oscar W (Elisabeth Vilhelmina Berg)  female  27.0      0   
9                Nasser, Mrs. Nicholas (Adele Achem)  female  14.0      1   

   Parch            Ticket     Fare Cabin Embarked  
0      0         A/5 21171   7.2500   NaN        S  
1      0          PC 17599  71.2833   C85        C  
2      0  STON/O2. 3101282   7.9250   NaN        S  
3      0            113803  53.1000  C123        S  
4      0            373450   8.0500   NaN        S  
5      0            330877   8.4583   NaN        Q  
6      0             17463  51.8625   E46        S  
7      1            349909  21.0750   NaN        S  
8      2            347742  11.1333   NaN        S  
9      0            237736  30.0708   NaN        C

# Preparing dataset

I apply the following transformations on the dataset.

1. Remove punctuations from the names. For example, "Braund, Mr. Owen Harris" becomes "Braund Mr Owen Harris".
2. Extract details from "Ticket". For example ticket "STON/O2. 3101282" becomes "STON/O2." and "3101282".

In [ ]:
def preprocess(df):
    df = df.copy()

    def normalize_name(x):
        return " ".join([v.strip(",()[].\"'") for v in x.split(" ")])

    def ticket_number(x):
        return x.split(" ")[-1]

    def ticket_item(x):
        items = x.split(" ")
        if len(items) == 1:
            return "NONE"
        return "_".join(items[0:-1])

    df["Name"] = df["Name"].apply(normalize_name)
    df["Ticket_number"] = df["Ticket"].apply(ticket_number)
    df["Ticket_item"] = df["Ticket"].apply(ticket_item)
    return df

preprocessed_train_df = preprocess(train_df)
preprocessed_serving_df = preprocess(serving_df)

preprocessed_train_df.head(5)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   

                                              Name     Sex   Age  SibSp  \
0                            Braund Mr Owen Harris    male  22.0      1   
1  Cumings Mrs John Bradley Florence Briggs Thayer  female  38.0      1   
2                             Heikkinen Miss Laina  female  26.0      0   
3         Futrelle Mrs Jacques Heath Lily May Peel  female  35.0      1   
4                           Allen Mr William Henry    male  35.0      0   

   Parch            Ticket     Fare Cabin Embarked Ticket_number Ticket_item  
0      0         A/5 21171   7.2500   NaN        S         21171         A/5  
1      0          PC 17599  71.2833   C85        C         17599          PC  
2      0  STON/O2. 3101282   7.9250   NaN        S       3101282    STON/O2.  
3      0            113803  53.1000  C123        S        113803        NONE  
4      0            373450   8.0500   NaN        S        373450        NONE

I only keep the list of the input features of the model.

In [ ]:
input_features = list(preprocessed_train_df.columns)
input_features.remove("Ticket")
input_features.remove("PassengerId")
input_features.remove("Survived")

print(f"Input features: {input_features}")

Input features: ['Pclass', 'Name', 'Sex', 'Age', 'SibSp', 'Parch', 'Fare', 'Cabin', 'Embarked', 'Ticket_number', 'Ticket_item']


# Converting Pandas dataset to TensorFlow Dataset

In [ ]:
def tokenize_names(features, labels=None):
    features["Name"] =  tf.strings.split(features["Name"])
    return features, labels

train_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_train_df,label="Survived").map(tokenize_names)
serving_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_serving_df).map(tokenize_names)

# Making an ensemble

I create 100 models with different seeds and combine their results. This approach removes a little bit the random aspects related to creating ML models.

In [ ]:
predictions = None
num_predictions = 0

for i in range(100):
    print(f"i:{i}")
    model = tfdf.keras.GradientBoostedTreesModel(
        verbose=0,
        features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
        exclude_non_specified_features=True,
        random_seed=i,
        honest=True,
    )
    model.fit(train_ds)

    sub_predictions = model.predict(serving_ds, verbose=0)[:,0]
    if predictions is None:
        predictions = sub_predictions
    else:
        predictions += sub_predictions
    num_predictions += 1

predictions/=num_predictions

kaggle_predictions = pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (predictions >= 0.5).astype(int)
    })

def make_submission(kaggle_predictions):
    path="/kaggle/working/submission.csv"
    kaggle_predictions.to_csv(path, index=False)
    print(f"Submission exported to {path}")

make_submission(kaggle_predictions)

i:0


[WARNING 24-02-15 18:42:13.2436 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:13.2447 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:13.2448 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:21.2039 UTC kernel.cc:1233] Loading model from path /tmp/tmpwmzfj7z2/model/ with prefix d0b7c4507bbb4e1f
[INFO 24-02-15 18:42:21.2096 UTC quick_scorer_extended.cc:903] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-02-15 18:42:21.2102 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-15 18:42:21.2103 UTC kernel.cc:1061] Use fast generic engine


i:1


[WARNING 24-02-15 18:42:23.3927 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:23.3927 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:23.3927 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:24.4786 UTC kernel.cc:1233] Loading model from path /tmp/tmpxaniaalm/model/ with prefix 4434c49ebe924357
[INFO 24-02-15 18:42:24.5001 UTC kernel.cc:1061] Use fast generic engine


i:2


[WARNING 24-02-15 18:42:24.9382 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:24.9382 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:24.9382 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:25.5646 UTC kernel.cc:1233] Loading model from path /tmp/tmpphirh4td/model/ with prefix 794a5885d1d64f65
[INFO 24-02-15 18:42:25.5694 UTC kernel.cc:1061] Use fast generic engine


i:3


[WARNING 24-02-15 18:42:26.0072 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:26.0073 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:26.0073 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:27.3464 UTC kernel.cc:1233] Loading model from path /tmp/tmprh9d4lc7/model/ with prefix 17ec439446f84753
[INFO 24-02-15 18:42:27.3760 UTC kernel.cc:1061] Use fast generic engine


i:4


[WARNING 24-02-15 18:42:27.8287 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:27.8288 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:27.8288 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:28.4778 UTC kernel.cc:1233] Loading model from path /tmp/tmp_ksqod7x/model/ with prefix fbff93c027464db4
[INFO 24-02-15 18:42:28.4838 UTC kernel.cc:1061] Use fast generic engine


i:5


[WARNING 24-02-15 18:42:28.9021 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:28.9021 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:28.9022 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:29.4815 UTC kernel.cc:1233] Loading model from path /tmp/tmpsd9h3e2u/model/ with prefix 81d718afc46444f5
[INFO 24-02-15 18:42:29.4849 UTC kernel.cc:1061] Use fast generic engine


i:6


[WARNING 24-02-15 18:42:29.9255 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:29.9256 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:29.9256 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:30.6469 UTC kernel.cc:1233] Loading model from path /tmp/tmpcybnpsfy/model/ with prefix f704b270eb5941cd
[INFO 24-02-15 18:42:30.6548 UTC kernel.cc:1061] Use fast generic engine


i:7


[WARNING 24-02-15 18:42:31.1055 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:31.1055 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:31.1056 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:32.3003 UTC kernel.cc:1233] Loading model from path /tmp/tmpcjpfsh76/model/ with prefix e8273960722342fe
[INFO 24-02-15 18:42:32.3240 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-15 18:42:32.3241 UTC kernel.cc:1061] Use fast generic engine


i:8


[WARNING 24-02-15 18:42:32.8588 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:32.8589 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:32.8589 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:33.6540 UTC kernel.cc:1233] Loading model from path /tmp/tmp7hj1q8d1/model/ with prefix aedb92d856a948fc
[INFO 24-02-15 18:42:33.6643 UTC kernel.cc:1061] Use fast generic engine


i:9


[WARNING 24-02-15 18:42:34.1105 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:34.1106 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:34.1106 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:35.0830 UTC kernel.cc:1233] Loading model from path /tmp/tmpnx01xyp7/model/ with prefix ce68cab3e8924a69
[INFO 24-02-15 18:42:35.1003 UTC kernel.cc:1061] Use fast generic engine


i:10


[WARNING 24-02-15 18:42:35.5279 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:35.5279 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:35.5279 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:36.2095 UTC kernel.cc:1233] Loading model from path /tmp/tmpdc2rjjbs/model/ with prefix 491dc76d27ec41d1
[INFO 24-02-15 18:42:36.2163 UTC kernel.cc:1061] Use fast generic engine


i:11


[WARNING 24-02-15 18:42:36.6810 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:36.6810 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:36.6810 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:37.6792 UTC kernel.cc:1233] Loading model from path /tmp/tmpwb2t4fjo/model/ with prefix c639ac311d3c499f
[INFO 24-02-15 18:42:37.6951 UTC kernel.cc:1061] Use fast generic engine


i:12


[WARNING 24-02-15 18:42:38.1414 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:38.1414 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:38.1414 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:38.8386 UTC kernel.cc:1233] Loading model from path /tmp/tmp4oktz156/model/ with prefix 333bd7a7931140e4
[INFO 24-02-15 18:42:38.8472 UTC kernel.cc:1061] Use fast generic engine


i:13


[WARNING 24-02-15 18:42:39.3225 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:39.3226 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:39.3226 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:40.1474 UTC kernel.cc:1233] Loading model from path /tmp/tmpf3u6r3k2/model/ with prefix 4485515d92fb4565
[INFO 24-02-15 18:42:40.1592 UTC kernel.cc:1061] Use fast generic engine


i:14


[WARNING 24-02-15 18:42:40.5803 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:40.5803 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:40.5803 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:41.2894 UTC kernel.cc:1233] Loading model from path /tmp/tmpy864v4xz/model/ with prefix 1839c1d4714242ce
[INFO 24-02-15 18:42:41.2960 UTC kernel.cc:1061] Use fast generic engine


i:15


[WARNING 24-02-15 18:42:42.3239 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:42.3239 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:42.3239 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:43.0648 UTC kernel.cc:1233] Loading model from path /tmp/tmpd363mffp/model/ with prefix e6c98d5ffd25468d
[INFO 24-02-15 18:42:43.0731 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-15 18:42:43.0731 UTC kernel.cc:1061] Use fast generic engine


i:16


[WARNING 24-02-15 18:42:43.5086 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:43.5086 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:43.5086 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:44.4233 UTC kernel.cc:1233] Loading model from path /tmp/tmp10ea2jiz/model/ with prefix be929906eec7493d
[INFO 24-02-15 18:42:44.4372 UTC kernel.cc:1061] Use fast generic engine


i:17


[WARNING 24-02-15 18:42:44.8470 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:44.8470 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:44.8470 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:45.7500 UTC kernel.cc:1233] Loading model from path /tmp/tmpznm3m7qo/model/ with prefix 1dbf8fb50afb4ca6
[INFO 24-02-15 18:42:45.7646 UTC kernel.cc:1061] Use fast generic engine


i:18


[WARNING 24-02-15 18:42:46.2033 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:46.2034 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:46.2034 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:47.0437 UTC kernel.cc:1233] Loading model from path /tmp/tmp6nel3rds/model/ with prefix cf3408adb2994094
[INFO 24-02-15 18:42:47.0566 UTC kernel.cc:1061] Use fast generic engine


i:19


[WARNING 24-02-15 18:42:47.5005 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:47.5005 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:47.5005 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:48.5583 UTC kernel.cc:1233] Loading model from path /tmp/tmp2tmm7_yz/model/ with prefix 8412679b584c42d5
[INFO 24-02-15 18:42:48.5783 UTC kernel.cc:1061] Use fast generic engine


i:20


[WARNING 24-02-15 18:42:49.6972 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:49.6973 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:49.6973 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:50.7099 UTC kernel.cc:1233] Loading model from path /tmp/tmpzmjeyuqb/model/ with prefix 94b0f2a9657e44fe
[INFO 24-02-15 18:42:50.7283 UTC kernel.cc:1061] Use fast generic engine


i:21


[WARNING 24-02-15 18:42:51.2102 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:51.2103 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:51.2105 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:51.9146 UTC kernel.cc:1233] Loading model from path /tmp/tmpk1zduz21/model/ with prefix fa465e0fda3c4da5
[INFO 24-02-15 18:42:51.9204 UTC quick_scorer_extended.cc:903] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-02-15 18:42:51.9209 UTC kernel.cc:1061] Use fast generic engine


i:22


[WARNING 24-02-15 18:42:52.3791 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:52.3792 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:52.3792 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:53.0739 UTC kernel.cc:1233] Loading model from path /tmp/tmptn7j2mlp/model/ with prefix 90054726248440ff
[INFO 24-02-15 18:42:53.0811 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-15 18:42:53.0812 UTC kernel.cc:1061] Use fast generic engine


i:23


[WARNING 24-02-15 18:42:53.5145 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:53.5145 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:53.5145 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:54.2939 UTC kernel.cc:1233] Loading model from path /tmp/tmptk4f7im7/model/ with prefix 18d6788c7f4a4c53
[INFO 24-02-15 18:42:54.3033 UTC kernel.cc:1061] Use fast generic engine


i:24


[WARNING 24-02-15 18:42:54.7542 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:54.7543 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:54.7543 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:55.4304 UTC kernel.cc:1233] Loading model from path /tmp/tmp0kkiu1wa/model/ with prefix d0192fb83159400d
[INFO 24-02-15 18:42:55.4369 UTC kernel.cc:1061] Use fast generic engine


i:25


[WARNING 24-02-15 18:42:55.8463 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:55.8464 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:55.8464 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:56.7107 UTC kernel.cc:1233] Loading model from path /tmp/tmpx_pta5ta/model/ with prefix 964b81be3d234811
[INFO 24-02-15 18:42:56.7229 UTC kernel.cc:1061] Use fast generic engine


i:26


[WARNING 24-02-15 18:42:57.1738 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:57.1738 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:57.1738 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:57.9916 UTC kernel.cc:1233] Loading model from path /tmp/tmpby8ru5cf/model/ with prefix 16a58448a1de436e
[INFO 24-02-15 18:42:58.0031 UTC kernel.cc:1061] Use fast generic engine


i:27


[WARNING 24-02-15 18:42:58.4162 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:58.4163 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:58.4163 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:42:59.1024 UTC kernel.cc:1233] Loading model from path /tmp/tmpkz2ro9nd/model/ with prefix efde61d43ed846c6
[INFO 24-02-15 18:42:59.1093 UTC kernel.cc:1061] Use fast generic engine


i:28


[WARNING 24-02-15 18:42:59.5510 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:59.5510 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:42:59.5511 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:00.1864 UTC kernel.cc:1233] Loading model from path /tmp/tmpwk4t6t2s/model/ with prefix 40541e946baf4260
[INFO 24-02-15 18:43:00.1920 UTC kernel.cc:1061] Use fast generic engine


i:29


[WARNING 24-02-15 18:43:00.6133 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:00.6133 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:00.6133 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:01.5583 UTC kernel.cc:1233] Loading model from path /tmp/tmpbmrafz2y/model/ with prefix 7789bbdf908e47b3
[INFO 24-02-15 18:43:01.5727 UTC kernel.cc:1061] Use fast generic engine


i:30


[WARNING 24-02-15 18:43:02.0467 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:02.0468 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:02.0468 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:03.4109 UTC kernel.cc:1233] Loading model from path /tmp/tmpv20tl4qg/model/ with prefix ce1a335605c14a37
[INFO 24-02-15 18:43:03.4394 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-15 18:43:03.4394 UTC kernel.cc:1061] Use fast generic engine


i:31


[WARNING 24-02-15 18:43:03.8892 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:03.8892 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:03.8893 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:04.7090 UTC kernel.cc:1233] Loading model from path /tmp/tmp6fr0sroe/model/ with prefix 709fb162daa64903
[INFO 24-02-15 18:43:04.7192 UTC kernel.cc:1061] Use fast generic engine


i:32


[WARNING 24-02-15 18:43:05.1464 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:05.1465 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:05.1465 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:05.7941 UTC kernel.cc:1233] Loading model from path /tmp/tmp_i6bpvy_/model/ with prefix 719a6082a64b44a9
[INFO 24-02-15 18:43:05.8005 UTC kernel.cc:1061] Use fast generic engine


i:33


[WARNING 24-02-15 18:43:06.2389 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:06.2389 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:06.2390 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:07.1326 UTC kernel.cc:1233] Loading model from path /tmp/tmpgsemlm1i/model/ with prefix de0fee8c62084bf0
[INFO 24-02-15 18:43:07.1464 UTC kernel.cc:1061] Use fast generic engine


i:34


[WARNING 24-02-15 18:43:07.9617 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:07.9618 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:07.9618 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:08.6970 UTC kernel.cc:1233] Loading model from path /tmp/tmphb5wbcf1/model/ with prefix e16db06fe4bf4bcd
[INFO 24-02-15 18:43:08.7063 UTC kernel.cc:1061] Use fast generic engine


i:35


[WARNING 24-02-15 18:43:09.6819 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:09.6821 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:09.6822 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:10.3959 UTC kernel.cc:1233] Loading model from path /tmp/tmpa1v0gaq_/model/ with prefix 6bf19dd097004485
[INFO 24-02-15 18:43:10.4042 UTC kernel.cc:1061] Use fast generic engine


i:36


[WARNING 24-02-15 18:43:10.8176 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:10.8176 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:10.8176 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:11.7945 UTC kernel.cc:1233] Loading model from path /tmp/tmp60212u7a/model/ with prefix 9591ba70fdb04bce
[INFO 24-02-15 18:43:11.8099 UTC kernel.cc:1061] Use fast generic engine


i:37


[WARNING 24-02-15 18:43:12.2353 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:12.2355 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:12.2356 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:13.0828 UTC kernel.cc:1233] Loading model from path /tmp/tmpf6u8euyy/model/ with prefix 20dc9647c10542c4
[INFO 24-02-15 18:43:13.0922 UTC kernel.cc:1061] Use fast generic engine


i:38


[WARNING 24-02-15 18:43:14.0444 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:14.0444 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:14.0444 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:14.9972 UTC kernel.cc:1233] Loading model from path /tmp/tmp3qub63ue/model/ with prefix d27593bf129f4a70
[INFO 24-02-15 18:43:15.0117 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-15 18:43:15.0118 UTC kernel.cc:1061] Use fast generic engine


i:39


[WARNING 24-02-15 18:43:15.4581 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:15.4581 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:15.4581 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:16.3916 UTC kernel.cc:1233] Loading model from path /tmp/tmphihc467q/model/ with prefix 77d69974fef34541
[INFO 24-02-15 18:43:16.4053 UTC kernel.cc:1061] Use fast generic engine


i:40


[WARNING 24-02-15 18:43:16.8498 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:16.8498 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:16.8498 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:17.4668 UTC kernel.cc:1233] Loading model from path /tmp/tmp5x0p3axz/model/ with prefix f0e9e33884aa4584
[INFO 24-02-15 18:43:17.4717 UTC kernel.cc:1061] Use fast generic engine


i:41


[WARNING 24-02-15 18:43:17.9102 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:17.9102 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:17.9102 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:18.9149 UTC kernel.cc:1233] Loading model from path /tmp/tmpqe1duj_y/model/ with prefix 1b6ba770eaa24657
[INFO 24-02-15 18:43:18.9320 UTC kernel.cc:1061] Use fast generic engine


i:42


[WARNING 24-02-15 18:43:19.3722 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:19.3722 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:19.3723 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:20.1517 UTC kernel.cc:1233] Loading model from path /tmp/tmp90w28dpw/model/ with prefix cc13f0915dda4a2b
[INFO 24-02-15 18:43:20.1621 UTC kernel.cc:1061] Use fast generic engine


i:43


[WARNING 24-02-15 18:43:20.6145 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:20.6145 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:20.6146 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:21.7024 UTC kernel.cc:1233] Loading model from path /tmp/tmp198qk8s6/model/ with prefix aae456ef2caf4fdd
[INFO 24-02-15 18:43:21.7218 UTC kernel.cc:1061] Use fast generic engine


i:44


[WARNING 24-02-15 18:43:22.1680 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:22.1680 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:22.1681 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:23.8550 UTC kernel.cc:1233] Loading model from path /tmp/tmp7cdg635u/model/ with prefix 0ae2113c5aae4c35
[INFO 24-02-15 18:43:23.8677 UTC quick_scorer_extended.cc:903] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-02-15 18:43:23.8692 UTC kernel.cc:1061] Use fast generic engine


i:45


[WARNING 24-02-15 18:43:24.3870 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:24.3871 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:24.3871 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:25.0026 UTC kernel.cc:1233] Loading model from path /tmp/tmpggmi6p9m/model/ with prefix 400147a68ec945d3
[INFO 24-02-15 18:43:25.0072 UTC kernel.cc:1061] Use fast generic engine


i:46


[WARNING 24-02-15 18:43:25.5303 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:25.5303 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:25.5303 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:26.5575 UTC kernel.cc:1233] Loading model from path /tmp/tmp_2juxvhc/model/ with prefix c571ba03b81343f5
[INFO 24-02-15 18:43:26.5765 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-15 18:43:26.5766 UTC kernel.cc:1061] Use fast generic engine


i:47


[WARNING 24-02-15 18:43:27.0375 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:27.0375 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:27.0376 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:27.9661 UTC kernel.cc:1233] Loading model from path /tmp/tmpf9ueltve/model/ with prefix 4d15c0882c034a92
[INFO 24-02-15 18:43:27.9810 UTC kernel.cc:1061] Use fast generic engine


i:48


[WARNING 24-02-15 18:43:28.4445 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:28.4445 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:28.4445 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:29.0791 UTC kernel.cc:1233] Loading model from path /tmp/tmptind88rj/model/ with prefix e8c92727055c460d
[INFO 24-02-15 18:43:29.0854 UTC kernel.cc:1061] Use fast generic engine


i:49


[WARNING 24-02-15 18:43:29.5401 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:29.5401 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:29.5401 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:30.2919 UTC kernel.cc:1233] Loading model from path /tmp/tmpibamcqrj/model/ with prefix 1c2182685e904a4d
[INFO 24-02-15 18:43:30.2996 UTC kernel.cc:1061] Use fast generic engine


i:50


[WARNING 24-02-15 18:43:30.7414 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:30.7415 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:30.7415 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:31.6958 UTC kernel.cc:1233] Loading model from path /tmp/tmp3f8ze1j_/model/ with prefix 4ba6f7321bf54c8d
[INFO 24-02-15 18:43:31.7102 UTC kernel.cc:1061] Use fast generic engine


i:51


[WARNING 24-02-15 18:43:32.1860 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:32.1861 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:32.1861 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:33.2417 UTC kernel.cc:1233] Loading model from path /tmp/tmpykxux7fc/model/ with prefix ac7697534ed648e2
[INFO 24-02-15 18:43:33.2603 UTC kernel.cc:1061] Use fast generic engine


i:52


[WARNING 24-02-15 18:43:33.7263 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:33.7264 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:33.7264 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:34.5558 UTC kernel.cc:1233] Loading model from path /tmp/tmpuyqej4q3/model/ with prefix 142ac1f109404789
[INFO 24-02-15 18:43:34.5646 UTC kernel.cc:1061] Use fast generic engine


i:53


[WARNING 24-02-15 18:43:34.9953 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:34.9954 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:34.9954 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:35.7245 UTC kernel.cc:1233] Loading model from path /tmp/tmpw49b0xn3/model/ with prefix 77396a956da7430a
[INFO 24-02-15 18:43:35.7328 UTC kernel.cc:1061] Use fast generic engine


i:54


[WARNING 24-02-15 18:43:36.1743 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:36.1744 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:36.1746 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:36.7860 UTC kernel.cc:1233] Loading model from path /tmp/tmpqme1rx1p/model/ with prefix 72f70cf9661f4739
[INFO 24-02-15 18:43:36.7898 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-15 18:43:36.7899 UTC kernel.cc:1061] Use fast generic engine


i:55


[WARNING 24-02-15 18:43:37.2221 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:37.2223 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:37.2224 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:38.1694 UTC kernel.cc:1233] Loading model from path /tmp/tmpeiy4kxgp/model/ with prefix e481c73d32894c31
[INFO 24-02-15 18:43:38.1848 UTC kernel.cc:1061] Use fast generic engine


i:56


[WARNING 24-02-15 18:43:38.6130 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:38.6130 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:38.6131 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:39.4177 UTC kernel.cc:1233] Loading model from path /tmp/tmpotos2wgy/model/ with prefix f5149c9ae8b943ef
[INFO 24-02-15 18:43:39.4289 UTC kernel.cc:1061] Use fast generic engine


i:57


[WARNING 24-02-15 18:43:39.8928 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:39.8929 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:39.8929 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:40.5158 UTC kernel.cc:1233] Loading model from path /tmp/tmptjqmfjtg/model/ with prefix 83dda47904364637
[INFO 24-02-15 18:43:40.5204 UTC kernel.cc:1061] Use fast generic engine


i:58


[WARNING 24-02-15 18:43:41.0031 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:41.0031 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:41.0032 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:41.7482 UTC kernel.cc:1233] Loading model from path /tmp/tmpn4xzwygv/model/ with prefix 0e74bc344fe54053
[INFO 24-02-15 18:43:41.7552 UTC kernel.cc:1061] Use fast generic engine


i:59


[WARNING 24-02-15 18:43:42.1733 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:42.1733 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:42.1733 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:42.9989 UTC kernel.cc:1233] Loading model from path /tmp/tmp1eueyox5/model/ with prefix 69db5732e61d49d9
[INFO 24-02-15 18:43:43.0093 UTC kernel.cc:1061] Use fast generic engine


i:60


[WARNING 24-02-15 18:43:43.9846 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:43.9847 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:43.9848 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:44.8578 UTC kernel.cc:1233] Loading model from path /tmp/tmpre6s9_by/model/ with prefix 32d5c0ee55254f7d
[INFO 24-02-15 18:43:44.8685 UTC kernel.cc:1061] Use fast generic engine


i:61


[WARNING 24-02-15 18:43:45.2957 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:45.2957 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:45.2958 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:45.9742 UTC kernel.cc:1233] Loading model from path /tmp/tmpej3yxxcb/model/ with prefix 38702d69cce14581
[INFO 24-02-15 18:43:45.9794 UTC kernel.cc:1061] Use fast generic engine


i:62


[WARNING 24-02-15 18:43:46.4309 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:46.4309 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:46.4309 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:47.3352 UTC kernel.cc:1233] Loading model from path /tmp/tmpv_vf1_qh/model/ with prefix f8c0a761de75481c
[INFO 24-02-15 18:43:47.3483 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-15 18:43:47.3484 UTC kernel.cc:1061] Use fast generic engine


i:63


[WARNING 24-02-15 18:43:47.7734 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:47.7734 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:47.7734 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:48.5558 UTC kernel.cc:1233] Loading model from path /tmp/tmp456qfctx/model/ with prefix 2782fe884735442b
[INFO 24-02-15 18:43:48.5657 UTC kernel.cc:1061] Use fast generic engine


i:64


[WARNING 24-02-15 18:43:48.9918 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:48.9918 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:48.9919 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:49.7292 UTC kernel.cc:1233] Loading model from path /tmp/tmpmr76fm8i/model/ with prefix 6a152da88e7f4934
[INFO 24-02-15 18:43:49.7377 UTC kernel.cc:1061] Use fast generic engine


i:65


[WARNING 24-02-15 18:43:50.1895 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:50.1895 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:50.1895 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:50.8606 UTC kernel.cc:1233] Loading model from path /tmp/tmp6316kjch/model/ with prefix 70bc0e495231462e
[INFO 24-02-15 18:43:50.8662 UTC kernel.cc:1061] Use fast generic engine


i:66


[WARNING 24-02-15 18:43:51.2960 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:51.2961 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:51.2961 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:52.0642 UTC kernel.cc:1233] Loading model from path /tmp/tmpiaq0v_0k/model/ with prefix 417c40ad9540411e
[INFO 24-02-15 18:43:52.0715 UTC kernel.cc:1061] Use fast generic engine


i:67


[WARNING 24-02-15 18:43:52.5169 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:52.5169 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:52.5169 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:53.6563 UTC kernel.cc:1233] Loading model from path /tmp/tmpncgq_m5k/model/ with prefix d68e168dd13a4f06
[INFO 24-02-15 18:43:53.6775 UTC kernel.cc:1061] Use fast generic engine


i:68


[WARNING 24-02-15 18:43:54.2935 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:54.2935 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:54.2935 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:55.1624 UTC kernel.cc:1233] Loading model from path /tmp/tmpnpmxr9rr/model/ with prefix 3ed61d23948842cc
[INFO 24-02-15 18:43:55.1744 UTC quick_scorer_extended.cc:903] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-02-15 18:43:55.1753 UTC kernel.cc:1061] Use fast generic engine


i:69


[WARNING 24-02-15 18:43:55.6423 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:55.6423 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:55.6424 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:56.3649 UTC kernel.cc:1233] Loading model from path /tmp/tmpcwttr0au/model/ with prefix 6075ce703c7a438b
[INFO 24-02-15 18:43:56.3722 UTC kernel.cc:1061] Use fast generic engine


i:70


[WARNING 24-02-15 18:43:56.8219 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:56.8219 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:56.8219 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:57.6587 UTC kernel.cc:1233] Loading model from path /tmp/tmp1rydk_vt/model/ with prefix dd29883360d84ddb
[INFO 24-02-15 18:43:57.6685 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-15 18:43:57.6686 UTC kernel.cc:1061] Use fast generic engine


i:71


[WARNING 24-02-15 18:43:58.1012 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:58.1013 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:43:58.1013 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:43:59.6048 UTC kernel.cc:1233] Loading model from path /tmp/tmpisgjz2mx/model/ with prefix 9712996cf60043df
[INFO 24-02-15 18:43:59.6151 UTC kernel.cc:1061] Use fast generic engine


i:72


[WARNING 24-02-15 18:44:00.1295 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:00.1295 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:00.1295 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:01.1949 UTC kernel.cc:1233] Loading model from path /tmp/tmp0ba8thxg/model/ with prefix 14cebd3e197f4704
[INFO 24-02-15 18:44:01.2186 UTC kernel.cc:1061] Use fast generic engine


i:73


[WARNING 24-02-15 18:44:01.9351 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:01.9352 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:01.9352 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:02.7102 UTC kernel.cc:1233] Loading model from path /tmp/tmpc4yy9b0e/model/ with prefix 4a416b051ad04957
[INFO 24-02-15 18:44:02.7192 UTC kernel.cc:1061] Use fast generic engine


i:74


[WARNING 24-02-15 18:44:03.7476 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:03.7476 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:03.7477 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:04.7117 UTC kernel.cc:1233] Loading model from path /tmp/tmpga7xbgcs/model/ with prefix 5d78b77c4d2c4a60
[INFO 24-02-15 18:44:04.7256 UTC kernel.cc:1061] Use fast generic engine


i:75


[WARNING 24-02-15 18:44:05.1730 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:05.1730 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:05.1730 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:05.9478 UTC kernel.cc:1233] Loading model from path /tmp/tmp5gyd40v8/model/ with prefix 88ccc34199a04ed5
[INFO 24-02-15 18:44:05.9569 UTC kernel.cc:1061] Use fast generic engine


i:76


[WARNING 24-02-15 18:44:06.9458 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:06.9459 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:06.9459 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:07.6169 UTC kernel.cc:1233] Loading model from path /tmp/tmp8_yhfjmx/model/ with prefix e1d2fe25802b4d03
[INFO 24-02-15 18:44:07.6218 UTC kernel.cc:1061] Use fast generic engine


i:77


[WARNING 24-02-15 18:44:08.0506 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:08.0506 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:08.0506 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:08.7237 UTC kernel.cc:1233] Loading model from path /tmp/tmpx3t47_rm/model/ with prefix f13e10b317c04be9
[INFO 24-02-15 18:44:08.7288 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-15 18:44:08.7288 UTC kernel.cc:1061] Use fast generic engine


i:78


[WARNING 24-02-15 18:44:09.1529 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:09.1529 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:09.1529 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:09.8396 UTC kernel.cc:1233] Loading model from path /tmp/tmpdp4bubda/model/ with prefix 2810ae3c682e42a0
[INFO 24-02-15 18:44:09.8469 UTC kernel.cc:1061] Use fast generic engine


i:79


[WARNING 24-02-15 18:44:10.2645 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:10.2645 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:10.2646 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:10.9597 UTC kernel.cc:1233] Loading model from path /tmp/tmp7qvmps6x/model/ with prefix acfb868e87bb4c73
[INFO 24-02-15 18:44:10.9676 UTC kernel.cc:1061] Use fast generic engine


i:80


[WARNING 24-02-15 18:44:11.4291 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:11.4292 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:11.4292 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:12.2940 UTC kernel.cc:1233] Loading model from path /tmp/tmptfemw3wg/model/ with prefix 1b07b77a12404785
[INFO 24-02-15 18:44:12.3044 UTC kernel.cc:1061] Use fast generic engine


i:81


[WARNING 24-02-15 18:44:12.7604 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:12.7604 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:12.7605 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:13.6750 UTC kernel.cc:1233] Loading model from path /tmp/tmpcv91odj5/model/ with prefix 2f1df60506ea42eb
[INFO 24-02-15 18:44:13.6876 UTC kernel.cc:1061] Use fast generic engine


i:82


[WARNING 24-02-15 18:44:14.5641 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:14.5641 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:14.5642 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:15.4031 UTC kernel.cc:1233] Loading model from path /tmp/tmpxqdmgzwz/model/ with prefix 923729716d694892
[INFO 24-02-15 18:44:15.4140 UTC kernel.cc:1061] Use fast generic engine


i:83


[WARNING 24-02-15 18:44:15.9141 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:15.9142 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:15.9142 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:16.7503 UTC kernel.cc:1233] Loading model from path /tmp/tmpjhh5fiaq/model/ with prefix 7829434fc68747d0
[INFO 24-02-15 18:44:16.7598 UTC kernel.cc:1061] Use fast generic engine


i:84


[WARNING 24-02-15 18:44:17.2060 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:17.2061 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:17.2061 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:18.2718 UTC kernel.cc:1233] Loading model from path /tmp/tmpt9aea0tf/model/ with prefix 6d1c9a743d9040d8
[INFO 24-02-15 18:44:18.2894 UTC kernel.cc:1061] Use fast generic engine


i:85


[WARNING 24-02-15 18:44:18.7312 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:18.7312 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:18.7312 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:19.4568 UTC kernel.cc:1233] Loading model from path /tmp/tmp27ukvrxu/model/ with prefix 1019e5ef9c8943a9
[INFO 24-02-15 18:44:19.4637 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-15 18:44:19.4638 UTC kernel.cc:1061] Use fast generic engine


i:86


[WARNING 24-02-15 18:44:19.8786 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:19.8787 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:19.8787 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:20.9094 UTC kernel.cc:1233] Loading model from path /tmp/tmpyc_jt3sh/model/ with prefix 8ab357fde06f4aa9
[INFO 24-02-15 18:44:20.9268 UTC kernel.cc:1061] Use fast generic engine


i:87


[WARNING 24-02-15 18:44:21.3601 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:21.3601 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:21.3601 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:22.5200 UTC kernel.cc:1233] Loading model from path /tmp/tmpr6y5vg31/model/ with prefix af12a9af13384b55
[INFO 24-02-15 18:44:22.5414 UTC kernel.cc:1061] Use fast generic engine


i:88


[WARNING 24-02-15 18:44:23.1809 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:23.1809 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:23.1809 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:24.0674 UTC kernel.cc:1233] Loading model from path /tmp/tmpd_dfqwn7/model/ with prefix fa2d3e8fc8c94e16
[INFO 24-02-15 18:44:24.0797 UTC kernel.cc:1061] Use fast generic engine


i:89


[WARNING 24-02-15 18:44:24.5157 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:24.5157 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:24.5157 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:25.1708 UTC kernel.cc:1233] Loading model from path /tmp/tmp2uzzln48/model/ with prefix 87f1b13c3846450c
[INFO 24-02-15 18:44:25.1761 UTC quick_scorer_extended.cc:903] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-02-15 18:44:25.1766 UTC kernel.cc:1061] Use fast generic engine


i:90


[WARNING 24-02-15 18:44:25.6061 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:25.6061 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:25.6061 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:26.4495 UTC kernel.cc:1233] Loading model from path /tmp/tmpmnzqopy2/model/ with prefix 094beb38a61a4a56
[INFO 24-02-15 18:44:26.4603 UTC kernel.cc:1061] Use fast generic engine


i:91


[WARNING 24-02-15 18:44:26.8914 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:26.8914 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:26.8915 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:27.6022 UTC kernel.cc:1233] Loading model from path /tmp/tmp_ar3t2dj/model/ with prefix 00a7769db9714b42
[INFO 24-02-15 18:44:27.6099 UTC kernel.cc:1061] Use fast generic engine


i:92


[WARNING 24-02-15 18:44:28.0375 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:28.0375 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:28.0375 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:29.1495 UTC kernel.cc:1233] Loading model from path /tmp/tmplp9h21t4/model/ with prefix 6af020d1a4c04159
[INFO 24-02-15 18:44:29.1711 UTC kernel.cc:1061] Use fast generic engine


i:93


[WARNING 24-02-15 18:44:29.8129 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:29.8129 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:29.8130 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:30.6691 UTC kernel.cc:1233] Loading model from path /tmp/tmpu7q8223g/model/ with prefix 179c1b30e1eb48fb
[INFO 24-02-15 18:44:30.6805 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-15 18:44:30.6805 UTC kernel.cc:1061] Use fast generic engine


i:94


[WARNING 24-02-15 18:44:31.1404 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:31.1404 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:31.1404 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:31.8904 UTC kernel.cc:1233] Loading model from path /tmp/tmp0ao08j2y/model/ with prefix 5f81ca50b80d4736
[INFO 24-02-15 18:44:31.8977 UTC kernel.cc:1061] Use fast generic engine


i:95


[WARNING 24-02-15 18:44:32.3270 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:32.3270 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:32.3270 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:33.1374 UTC kernel.cc:1233] Loading model from path /tmp/tmpz90oegj9/model/ with prefix 87877f66622548e0
[INFO 24-02-15 18:44:33.1463 UTC kernel.cc:1061] Use fast generic engine


i:96


[WARNING 24-02-15 18:44:33.5576 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:33.5576 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:33.5577 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:34.4482 UTC kernel.cc:1233] Loading model from path /tmp/tmp5bunutzl/model/ with prefix d8cc2a4b859045d0
[INFO 24-02-15 18:44:34.4594 UTC kernel.cc:1061] Use fast generic engine


i:97


[WARNING 24-02-15 18:44:34.8830 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:34.8830 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:34.8831 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:35.5960 UTC kernel.cc:1233] Loading model from path /tmp/tmp20l0g1r1/model/ with prefix c75e2c95a1274b04
[INFO 24-02-15 18:44:35.6019 UTC kernel.cc:1061] Use fast generic engine


i:98


[WARNING 24-02-15 18:44:36.6798 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:36.6798 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:36.6798 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:37.4296 UTC kernel.cc:1233] Loading model from path /tmp/tmp9n733r1i/model/ with prefix 8b7d90d369af49be
[INFO 24-02-15 18:44:37.4383 UTC kernel.cc:1061] Use fast generic engine


i:99


[WARNING 24-02-15 18:44:37.8744 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:37.8744 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-15 18:44:37.8744 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-15 18:44:38.9101 UTC kernel.cc:1233] Loading model from path /tmp/tmpyx19834h/model/ with prefix b66da17f78c34a19
[INFO 24-02-15 18:44:38.9249 UTC kernel.cc:1061] Use fast generic engine


Submission exported to /kaggle/working/submission.csv
